src/pages/login/Login.scss

```
@@ -1,3 +1,58 @@
-div{
-    font-size: 15px;
-}
\ No newline at end of file
+.login {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+
+  .title {
+    text-align: center;
+    margin: 20px;
+    font-size: 24px;
+    font-weight: 700;
+    color: black;
+    margin-top: 60px;
+  }
+  .inputLabel {
+    margin-top: 60px;
+    font-size: 18px;
+    font-weight: 700;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 10px;
+  }
+
+  .input {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 312px;
+    margin-left: auto;
+    margin-right: auto;
+
+    .inputData {
```

```suggestion
  .input {
    display: flex;
    flex-direction: column;
    justify-content: center;
    width: 312px;
    margin-left: auto;
    margin-right: auto;

    &Data {
```
여기서 굳이 적용하진 않아도 될 것 같지만,
클래스 명에서 중복되는 내용은 [부모선택자](https://sass-lang.com/documentation/style-rules/parent-selector)를 이용해 축약할 수 있어요!

---

아하 감사합니다. 수정완료 했습니다.